README.md

```diff
@@ -19,6 +19,14 @@ None.
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
 | clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |
+| clamav_scan_copy | Whether to copy infected files to quarantine folder. | false | No |
+| clamav_scan_exclude_directories | Directory trees to exclude from scan operations. | `{ /dev, /proc, /sys, /var/spool/clamav }` | No |
```

Comment:
This is `list`, not a `dict`:
```suggestion
| clamav_scan_exclude_directories | A list of regexes matching irectory trees that are to be excluded from scan operations. | `[^/dev, ^/proc, ^/sys, ^/var/spool/clamav]` | No |
```

I also updated the default values to match those in `defaults/main.yml` and tweaked the variable description to mention that the list entries are regexes.